drivers/dma/dma_mcux_lpc.c

```diff
@@ -312,13 +345,53 @@ static int dma_mcux_lpc_configure(const struct device *dev, uint32_t channel,
 			data->descriptor_index = data->descriptor_used;
 		}
 	} else {
```

Comment:
the code for the chaining set up being in the else block of the scatter/gather implies that the two features are mutually exclusive, is this the case for LPC DMA?

Reply:
I don't think this comment is relevant anymore. Please correct me if I am wrong

Reply:
yes it looks like these two functionalities have been separated now in one of the recent pushes and are no longer mutually exclusive in the code, which is as expected